test/server_integration/http/cache/index.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }) {
```

Comment:
NIT: file could probably be in TS instead of JS.

Reply:
I explored this, but in order to get really any benefit from the conversion, we also need to convert all of the `server_integration` services as well, and the type definitions for supertest are out of date (and essentially EOL), which is the only real external dependency, so I decided not to do the conversion as part of this PR